nexus/src/app/disk.rs

```diff
@@ -456,4 +400,277 @@ impl super::Nexus {
 
         Ok(())
     }
+
+    /// Import blocks from a URL into a disk
+    pub async fn import_blocks_from_url_for_disk(
+        self: &Arc<Self>,
+        opctx: &OpContext,
+        disk_selector: &params::DiskSelector,
+        params: params::ImportBlocksFromUrl,
+    ) -> UpdateResult<()> {
+        let authz_disk: authz::Disk;
+
+        (_, _, authz_disk) = self
```

Comment:
```suggestion
        (.., authz_disk) = self
```

Doesn't seem like you need to explicitly destruct these values if they're not being used.

Reply:
right, addressed in 06471b9a